### PR TITLE
[OSJC-199] Fixing some issues found by coverity scan

### PR DESCRIPTION
**Pull request by corey112358**
High: Resource Leak
/src/main/java/com/openshift/client/SSHPublicKey.java

Medium: EC: Comparing incompatible types for equality
/src/main/java/com/openshift/client/Message.java

Medium: Arguments in wrong order
/src/main/java/com/openshift/client/ConnectionBuilder.java

Medium: Dereference null return value
/src/main/java/com/openshift/client/HttpMethod.java

Low: Checking string value using == or ! =
/src/main/java/com/openshift/client/HttpMethod.java

See JIRA for more information

### CODE REVIEW

**Comment by corey112358** [comment]
[test]

**Comment by corey112358** [comment]
@adietish Can you give this a check when you have a chance and merge if it looks ok?

**Comment by jcantrill** [comment]
LGTM.  merge at your leisure.

**Comment by corey112358** [comment]
[merge]

**Comment by adietish** [comment]
@corey112358 great, thx!

File: src/main/java/com/openshift/client/ConnectionBuilder.java [hunk 1]
```diff
@@ -97,8 +97,8 @@ public IOpenShiftConnection create() {
 					clientId,
 					null,
 					null,
-					authIV,
 					authKey,
+					authIV,
```
Comment (adietish) [inline src/main/java/com/openshift/client/ConnectionBuilder.java]: good catch!

**Pull request merged**